blurp-app/src/components/searchbar.jsx

```diff
@@ -3,8 +3,15 @@
 function SearchBar() {
     return (
         <>
-            <input type="search">
-            </input>
+            <form class="absolute w-full flex items-center justify-center m-1">   
+            <div class="relative w-3/5">
+                <div class="absolute inset-y-0 left-[10px] flex items-center pl- pointer-events-none">
+                    <svg aria-hidden="true" class="w-5 h-5 text-gray-500 dark:text-gray-400" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z" clip-rule="evenodd"></path></svg>

```

Comment:
For longer lines of code like this, think about how to format it nicely for readability